File: Codechef/Winnerr/Winnerr.py
T = int(input())
while T:
    T -= 1
    Pa, Pb, Qa, Qb = map(int, input().split())
    max_P = max(Pa, Pb)
    max_Q = max(Qa, Qb)
    if (max_P > max_Q):
        print("Q")
    elif (max_P < max_Q):
        print("P")
    else:
        print("TIE")

File: Codechef/Winnerr/Winnerr.cpp
#include <iostream>
using namespace std;

int main()
{
    int T;
    cin >> T;
    while (T--)
    {
        int Pa, Pb, Qa, Qb, max_P, max_Q;
        cin >> Pa >> Pb >> Qa >> Qb;
        max_P = max(Pa, Pb);
        max_Q = max(Qa, Qb);
        if (max_P > max_Q)
        {
            cout << "Q" << end;
        }
        else if (max_P < max_Q)
        {
            cout << "P" << end;
        }
        else
        {
            cout << "TIE" << end;
        }
        
    }
    return 0;
}